hack/libsubid_tag.sh

```diff
@@ -24,6 +24,3 @@ int main() {
 	return 0;
 }
 EOF
-if test $? -eq 0 ; then
-	echo libsubid
-fi
```

Comment:
Could you move that into a separate commit?  We probably need to cherry-pick the OOM fix.

Reply:
no that is some debug change that should not be committed